subsys/net/ip/tcp.c

```diff
@@ -1369,6 +1570,15 @@ static struct tcp *tcp_conn_alloc(struct net_context *context)
 	k_work_init_delayable(&conn->recv_queue_timer, tcp_cleanup_recv_queue);
 	k_work_init_delayable(&conn->persist_timer, tcp_send_zwp);
 
+#if defined(CONFIG_NET_TCP_KEEPALIVE)
+	conn->keep_alive = false;
+	conn->keep_idle = CONFIG_NET_TCP_KEEPIDLE_DEFAULT;
+	conn->keep_intvl = CONFIG_NET_TCP_KEEPINTVL_DEFAULT;
+	conn->keep_cnt = CONFIG_NET_TCP_KEEPCNT_DEFAULT;
+	NET_INFO("keepalive timer init idle = %d, interval = %d, cnt = %d", conn->keep_idle, conn->keep_intvl, conn->keep_cnt);
```

Comment:
Not sure if this needs to be in info level, would user be interested to see this always. I would suggest `NET_DBG` here instead.